pysus/tests/test_data/test_sinan.py

```diff
@@ -99,10 +98,6 @@ def test_type_convertion(self):
         self.assertTrue(self.dbf_name.exists())
         assert not all(df.dtypes == "object")
 
-    def test_geocode(self):
-        self.assertTrue(self.data_pickle.exists())
```

Comment:
I didn't see much use for this test..

Reply:
I don't remember if at some point sinan generates this pickle file, but from the name of the test it seems that this test is incomplete.